### PR TITLE
Switch to go version of sentry-kubernetes with first m…

### PR DESCRIPTION
**Pull request by matias-easymile**
…inimal version of the chart

I didn't implemented volume mount for the kubeconfig.
I think this need to be passed using a secret and we can pass it like SENTRY_DSN.

### CODE REVIEW

File: charts/sentry-kubernetes/README.md
Comment (Mokto) [inline charts/sentry-kubernetes/README.md:1]: Can you please remove your changes here please ?

File: charts/sentry-kubernetes/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: sentry-kubernetes
 description: A Helm chart for sentry-kubernetes
   (https://github.com/getsentry/sentry-kubernetes)
 type: application
-version: 0.3.4
+version: 0.4.0
```
Comment (Mokto) [inline charts/sentry-kubernetes/Chart.yaml]: This is done automatically.

File: charts/sentry-kubernetes/README.md
Comment (matias-easymile) [reply charts/sentry-kubernetes/README.md:1]: Fixed

Comment (matias-easymile) [reply charts/sentry-kubernetes/README.md:1]: You mean entire modification in readme.md?

Comment (Mokto) [reply charts/sentry-kubernetes/README.md:1]: The auto formatting.

**Review by Mokto — APPROVED** [review]

**Pull request merged**